sidebar.js

```diff
@@ -133,6 +133,20 @@ const sidebars = {
       items: [{ type: "autogenerated", dirName: "scripting/trust_scripts" }],
     },
   ],
+  jargonSidebar: [
+    {
```

Comment:
edited label, title, description, slug and dirName to point to jargon when there's a right spot for it.